weblate/addons/models.py

```diff
@@ -207,198 +207,97 @@ def handle_addon_error(addon, component):
     if not addon.addon.can_install(component, None):
         addon.disable()
 
+def handle_addon_event(component, event_type, addon_method, description):
+    addons = Addon.objects.filter_event(component, event_type)
 
-@receiver(vcs_pre_push)
-def pre_push(sender, component, **kwargs):
-    for addon in Addon.objects.filter_event(component, EVENT_PRE_PUSH):
-        component.log_debug("running pre_push add-on: %s", addon.name)
+    for addon in addons:
+        component.log_debug(f"running {event_type} add-on: {addon.name}")
```

Comment:
Use event name instead of ID (which is integer):

```suggestion
        component.log_debug(f"running {EVENT_NAMES[event_type]} add-on: {addon.name}")
```